prusti-tests/tests/verify/pass/issues/issue-1215-1.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: -Pcheck_overflows=false
```

Comment:
As stated in a previous comment, these `compile-flags` should be removed, since you placed the tests in `verify` and not `verify_overflow`.

Reply:
Resolved in #[5c95497](https://github.com/viperproject/prusti-dev/pull/1281/commits/5c954970094fa257cc4ae497580d187895d13337)